$OpenBSD: patch-include_mbedtls_config_h,v 1.14 2021/03/12 17:41:04 bket Exp $

MBEDTLS_THREADING_PTHREAD and MBEDTLS_THREADING_C are required by
www/hiawatha.

Index: include/mbedtls/config.h
--- include/mbedtls/config.h.orig
+++ include/mbedtls/config.h
@@ -1783,7 +1783,7 @@
  *
  * Uncomment this to enable pthread mutexes.
  */
-//#define MBEDTLS_THREADING_PTHREAD
+#define MBEDTLS_THREADING_PTHREAD
 
 /**
  * \def MBEDTLS_VERSION_FEATURES
@@ -2979,7 +2979,7 @@
  *
  * Enable this layer to allow use of mutexes within mbed TLS
  */
-//#define MBEDTLS_THREADING_C
+#define MBEDTLS_THREADING_C
 
 /**
  * \def MBEDTLS_TIMING_C
